README.md

```diff
@@ -1,2 +1,151 @@
 # enumerable
+
+<a name="readme-top"><h1 align="center">Enumerable</h1></a>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+   - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 enumerable<a name="about-project"></a>
+
 enumerable is a Ruby module that provides a set of methods that allow objects to be iterated and processed in various ways, such as sorting, finding, and grouping their elements.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+<details>
+<summary>Ruby</summary>
+  <ul>
+    <li><a href="https://rubygems.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
```

Comment:
- [OPTIONAL] Kindy remove from [PostgreSQL](https://rubygems.org/) Ruby Tech Stack section, We did not use it.

![image](https://github.com/theodoku/enumerable/assets/79058364/5bea3472-1218-45f1-ba2a-0c924b8d6646)

Reply:
PostgreSQL removed